hack/tools/go.mod

```diff
@@ -9,13 +9,12 @@ require (
 	github.com/google/gofuzz v1.2.0 // indirect
 	github.com/imdario/mergo v0.3.11 // indirect
 	github.com/itchyny/gojq v0.12.2
-	github.com/onsi/ginkgo v1.16.1
+	github.com/onsi/ginkgo v1.16.4
 	github.com/onsi/gomega v1.11.0 // indirect
```

Comment:
gomega v1.13.0 is specified in go.mod. we can use gomega v1.13.0 here.

Reply:
I just dropped all indirect dependencies. Apparently `go mod tidy` doesn't drop them by itself.

Reply:
Thanks updating.